CMakeLists.txt

```diff
@@ -44,11 +44,13 @@ set(TEST_LIBS ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} ${Boost_LIBRARIES})
 
 # Workaround
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.5.0)
-  else()
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 3.5.0)
```

Comment:
@Fraser999 

> We'd need to build CRUX as a standalone project to have the tests hit this bit, so I think @breese's concern is valid. Also, checking for versions being equal to a specific version rather than less/greater than leaves the possibility for errors. (e.g. how would we handle v3.4.1?)

I do not oppose, it is just that we'll not know until we test it. Then we can update the CMakeList.txt file.